plugins/ipam/host-local/backend/allocator/config.go

```diff
@@ -113,9 +115,10 @@ func LoadIPAMConfig(bytes []byte, envArgs string) (*IPAMConfig, string, error) {
 	numV6 := 0
 	for i, _ := range n.IPAM.Ranges {
 		if err := n.IPAM.Ranges[i].Canonicalize(); err != nil {
-			return nil, "", fmt.Errorf("Cannot understand range %d: %v", i, err)
+			return nil, "", fmt.Errorf("invalid range set %d: %s", i, err)
 		}
-		if len(n.IPAM.Ranges[i].RangeStart) == 4 {
+
+		if len(n.IPAM.Ranges[i][0].RangeStart) == 4 {
```

Comment:
Maybe we should change this to To4()... I know we canonicalize the start so it'll be len=4 for v4, but I don't think we use len() anywhere else for v4/v6 checks (though we do use it later for canonicalization and range checking, but only to compare lengths to each other).